hack/build/ctr/cloudbuild.yaml

```diff
@@ -0,0 +1,28 @@
+# simple cloudbuild to build github.com/contianerd/containerd/cmd/ctr
```

Comment:
I don't expect anyone else to need to use `hack/build/ctr/...`, but in the interests of transparency I'm attempting to create a reproducible (?) cross-platform build of `ctr` with the tiny patch we are waiting on for use in the base image, and NOT build it from my workstation :^)

Reply:
also: we will stop using these once we can get this patch packaged upstream, or we will figure something else out. 

being able to use this patch now means we can do image import properly without errors and handle errors.